server/server.js

```diff
@@ -17,6 +44,42 @@ app.get('/carousel/:gameId', (req, res) => {
   });
 });
 
+// Update
+
+app.put('/put', (req, res) => {
+  let newRecord = {
+    gameTitle: faker.name.title(),
+    newPrice: 59.99,
+    preOwnedPrice: 54.99,
+    digitalPrice: 59.99,
+    mainImage: seed.randomMainImage(),
+    images: seed.randomImages(),
+  }
```

Comment:
For our purposes this should work just fine, but it would probably be better to generate the new record outside and pass it in through the request body. It would make this more usable in a real world context.  As is, it doesn't allow you to create a new record with specific data, just randomized faker data.

Reply:
Thank you for pointing this out. I was going to do this, but I figured it would make more sense to just leave this here so I know what the data needs to look like when I build up the front end.